geomstats/_backend/tensorflow/__init__.py

```diff
@@ -341,11 +365,19 @@ def _assignment_single_value(x, value, indices, axis=0):
         Copy of x where value was assigned at all indices (and possibly
         along an axis).
     """
+    if _is_boolean(indices):
+        indices = [index for index, val in enumerate(indices) if val]
```

Comment:
This line comes up often, and could be factorize into a `_from_bools_to_indices` aux function.

Reply:
True ! Do you think this should stay here, or could be useful elsewhere and go to the backend ? (In which case, a subsequent PR could be appropriate.)